src/webob/acceptparse.py

```diff
@@ -686,10 +686,16 @@ def basic_filtering(self, language_tags):
         suggested as a matching scheme for the ``Accept-Language`` header in
         :rfc:`RFC 7231, section 5.3.5 <7231#section-5.3.5>`, and defined in
         :rfc:`RFC 4647, section 3.3.1 <4647#section-3.3.1>`. It filters the
-        tags in the `language_tags` argument, and returns a list of the tags
-        that match the language ranges in the header according to the Basic
-        Filtering matching scheme, in descending order of preference, together
-        with the qvalue of the range each tag matched.
+        tags in the `language_tags` argument and returns the ones that match
+        the header according to the matching scheme. The returned list is a
+        list of (language tag, qvalue) tuples, in descending order of qvalue;
+        if two or more tags have the same qvalue, they are returned in the same
+        order as the order in the header of the ranges they matched. If the
+        matched range is the same for two or more tags (i.e. their matched
+        ranges have the same qvalue and the same position in the header), then
+        they are returned in the same order as their order in the
+        `language_tags` argument. (If `language_tags` is unordered, e.g. if it
+        is a set or a dict, then that order may not be reliable.)
```

Comment:
Remove parentheses, as this is really not a parenthetical statement, but a sentence of distinct importance.